articles/time_to_say_goodby.md

```diff
@@ -0,0 +1,55 @@
+ 
+On the other hand, transfer to operation is not proceeding because the project team is still receiving all new requests and tries to handle these. 
+ 
+And as there might be a certain kind of uncertainty in terms of how things are going and how much longer we will be involved in the project, people are happy to be contacted and needed.
+
+Those situations lead to the fact, that knowledge transfer and handover does not take place, but resentment due to unclear responsibilities takes over.
+
+Sometimes the project manager simply says there is no more work and there is no information flow any longer. Project members feel dropped, not appreciated and some kind of frustration sinks in. 
+ 
+By the way, something similar might happen when project members are assigned to a new project and expectations are not aligned between project members, former and new project managers.
+ 
+And last but not least a completely different aspect: people are getting frustrated because of all the learnings they did within the project, realizing, that in the end the new project is making exactly the same mistakes right at the beginning.
+ 
+Based on these insights I would like to give you 6 suggestions of what can be done to reduce or avoid the previously mentioned situations.
+ 
+1. Involve Operations and support as soon as possible
+Here I do not mean just invite them to the team meeting, but also ask them what their expectations to the project are and what they need in order to take over responsibility at the end of the project. Ask them for document reviews and involve them as tester. Create an informal documentation of all issues and decisions taken as further reference and handbook. 
+ 
+2. Give the transition phase a name, define, plan and communicate this phase
+Just like the development must be planned beforehand  also the transition needs to be planned in advance. Official communication that this phase will now start and what will be the achievements of this phase will help all involved people and give some kind of orientation.
+ 
+3. Redefine roles, responsibilities and meetings in the team and communicate the result
+With the new phase and activities, responsibilities will change, too. Maybe a business analyst will no longer be responsible for requirements only but also for training and support of the support team or a test manager will no longer be needed. Some tasks shall be done by operations now and no longer by the project team itself. Therefore, a new definition of roles and responsibilities in the team is crucial as well as communication of the new assignments. Team meetings or defect meetings may no longer be required. Therefore, meetings to discuss existing production issues, progress within trainings or performance and usage statistics are more relevant. From my perspective it is better to stop old meeting series and setup new meetings with new agenda and participants instead of reusing old ones to have an official shift.
```

Comment:
Klarer: "With the new phase and activities, responsibilities will change, too." -> "With the new phase and activities, responsibilities will change".